vg/font.go

```diff
@@ -237,12 +237,9 @@ func fontData(name string) ([]byte, error) {
 // If the first font file found is unreadable or cannot be parsed, then
 // subsequent directories are not tried, and the font will fail to load.
 //
-// The default slice contains, in the following order, the values of the
-// environment variable VGFONTPATH if it is defined, then the vg
-// source fonts directory if it is found (i.e., if vg was installed by
-// go get).  If the resulting FontDirs slice is empty then the current
-// directory is added to it.  This slice may be changed to load fonts
-// from different locations.
+// The default slice contains the values of the environment variable
```

Comment:
I would say that it is initialised with the contents of `VGFONTPATH` is it is defined rather than this; it's exported and mutable.

Reply:
Sorry, that last clause was a justification of my comment. The point was that it may not necessarily contain the values, since because it is exported and mutable, it may have been changed.

```
// The default slice is initialised with the contents of the VGFONTPATH
// environment variable if it is defined.
// This slice may be changed to load fonts from different locations.
```

Reply:
ok. done.